// Webezy.io Generated proto DO NOT EDIT
syntax = "proto3";

import "SamplePackage.proto";
import "OtherPackage.proto";


service SampleService {
	// [webezyio] - SampleRPC - Some description for RPC
	rpc SampleRPC (domain.SamplePackage.v1.SampleMessage) returns (domain.SamplePackage.v1.SampleMessage);
	// [webezyio] - TestRPC - Some description for RPC
	rpc TestRPC (stream domain.OtherPackage.v1.OtherMessage) returns (domain.OtherPackage.v1.OtherMessage);
	// [webezyio] - OtherRPC - Some description for RPC
	rpc OtherRPC (stream domain.OtherPackage.v1.OtherMessage) returns (domain.SamplePackage.v1.SampleMessage);
}


